### PR TITLE
WIP: AWS Bootstrap for HBase and FSDS

### PR DESCRIPTION
**Pull request by aheyne**
Signed-off-by: Austin Heyne <aheyne@ccri.com>

### CODE REVIEW

File: geomesa-spark/geomesa-spark-jts/src/main/scala/org/apache/spark/sql/jts/JTSTypes.scala [hunk 1]
```diff
@@ -75,11 +75,25 @@ private [spark] class GeometryUDT extends AbstractGeometryUDT[Geometry]("geometr
...
 }
-
 case object GeometryUDT extends GeometryUDT
 
 private [spark] class GeometryCollectionUDT
   extends AbstractGeometryUDT[GeometryCollection]("geometrycollection")
-
 object GeometryCollectionUDT extends GeometryCollectionUDT
 
+object Test {
```
Comment (elahrvivaz) [inline geomesa-spark/geomesa-spark-jts/src/main/scala/org/apache/spark/sql/jts/JTSTypes.scala]: is this supposed to be part of the commit, or was it just left over?

File: geomesa-hbase/geomesa-hbase-dist/pom.xml [hunk 2]
```diff
@@ -21,7 +21,10 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <gmtools.dist.name>GEOMESA_HBASE</gmtools.dist.name> <!-- used in tools filtering -->
+        <!-- used in tools filtering -->
+        <gmtools.dist.name>GEOMESA_HBASE</gmtools.dist.name>
+        <gmtools.assembly.name>geomesa-hbase_${scala.binary.version}-${project.version}</gmtools.assembly.name>
+        <geopyspark.version>0.3.0.gm.6</geopyspark.version>
```
Comment (elahrvivaz) [inline geomesa-hbase/geomesa-hbase-dist/pom.xml]: seems like this isn't used?

File: geomesa-hbase/geomesa-hbase-dist/src/main/assemblies/component.xml [hunk 3]
```diff
@@ -38,9 +38,10 @@
             <fileMode>0644</fileMode>
             <useProjectArtifact>false</useProjectArtifact>
             <useTransitiveDependencies>false</useTransitiveDependencies>
-            <useStrictFiltering>true</useStrictFiltering>
+            <!--<useStrictFiltering>true</useStrictFiltering>-->
```
Comment (elahrvivaz) [inline geomesa-hbase/geomesa-hbase-dist/src/main/assemblies/component.xml]: we should evaluate what the different jars are that get included with this change

File: geomesa-hbase/geomesa-hbase-spark-runtime/pom.xml [hunk 4]
```diff
@@ -88,6 +88,13 @@
             <artifactId>geomesa-tools_${scala.binary.version}</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.protobuf</groupId>
+            <artifactId>protobuf-java</artifactId>
+            <version>2.5.0</version>
+            <scope>runtime</scope>
+        </dependency>
```
Comment (elahrvivaz) [inline geomesa-hbase/geomesa-hbase-spark-runtime/pom.xml]: do we need this change? we should run the cqs script to make sure nothing changed

**Comment by elahrvivaz** [comment]
would it be possible to have common versions of any of the scripts? i didn't look at them very closely, but seems like there might be a fair amount of duplication

**Comment by aheyne** [comment]
Yes, a good chunk could probably be pulled out to geomesa-tools.

**Comment by elahrvivaz** [comment]
I'm going to close this for now, we can re-open if we ever go back to it.

**Pull request closed without merging**